README.txt

```diff
@@ -1,162 +1,89 @@
+INTRODUCTION
+------------
```

Comment:
Moving from asciidoc to markdown may be a good idea anyway, since it has proven to be more popular.